tests/dashbio_demos/app_circos.py

```diff
@@ -599,817 +1237,86 @@ def update_output(
     # Return Circos Graph with specified layout & dataset
     @app.callback(
         Output("circos-hold", "children"),
-        [
-            Input("circos-tabs", "value"),
-            Input("circos-selector", "value"),
-            Input("size-slider", "value"),
-            Input("size-slider-custom", "value"),
-            Input("init", "n_intervals"),
-            Input("render-button", "n_clicks"),
-            Input("data-table", "selected_rows"),
-        ],
+        [Input('circos-preloaded-uploaded', 'value'),
+         Input('circos-graph-type', 'value'),
+         Input("circos-size", "value"),
+         Input("render-button", "n_clicks"),
+         Input("data-table", "selected_rows"),
+         Input('circos-tabs', 'value')],
         [
             State("output-data-upload", "children"),
             State("data-table", "data"),
-            State("data-selector", "value"),
+            State("circos-view-dataset", "value"),
         ],
     )
-    def init(
-            tabs,
+    def show_circos_graph(
+            pre_up,
             circos_select,
             size,
-            size_custom,
-            init_onstart,
             render_button,
-            selected_row,
+            selected_row, _,
```

Comment:
Shouldn't `_` go on its own line, out of consistency?

Reply:
@mkcor Fixed in 9b2847a :)